pkg/agent/agent.go

```diff
@@ -503,7 +503,7 @@ func (i *Initializer) initNodeLocalConfig() error {
 	}
 
 	if i.networkConfig.TrafficEncapMode.IsNetworkPolicyOnly() {
-		i.nodeConfig = &config.NodeConfig{Name: nodeName, NodeIPAddr: localAddr, BridgeName: i.ovsBridgeClient.GetBridgeName(), UplinkNetConfig: new(config.AdapterNetConfig)}
+		i.nodeConfig = &config.NodeConfig{Name: nodeName, NodeIPAddr: localAddr, UplinkNetConfig: new(config.AdapterNetConfig)}
```

Comment:
Add OVSBridge?

Reply:
Good catch! This must have caused some issues in networkPolicyOnly mode too.